packages/ladle/package.json

```diff
@@ -28,7 +28,7 @@
     "clean": "rimraf dist && rimraf .ladle && rimraf build && rimraf *.tsbuildinfo",
     "serve": "node ./lib/cli/cli.js serve",
     "test": "cross-env IMPORT_ROOT=\"./\" vitest",
-    "types": "tsc --project tsconfig.typesoutput.json"
+    "types": "tsc --project tsconfig.typesoutput.json && cp ./lib/app/exports.d.ts ./lib/app/exports.d.cts"
```

Comment:
`cp` won't work across platforms (e.g. windows) - would you like me to include something like https://www.npmjs.com/package/cpy-cli so that it does, or are you ok with this as it will generally only run in CI/CD processes?

Reply:
yea, this doesn't need to be x-platform